action-server/covidflow/setup_structlog.py

```diff
@@ -84,6 +97,14 @@ def _add_record(log_record, key, value):
 class CustomJsonFormatter(jsonlogger.JsonFormatter):
     def add_fields(self, log_record, record, message_dict):
         super(CustomJsonFormatter, self).add_fields(log_record, record, message_dict)
+
+        if log_record.get(IS_STRUCTLOG_KEY) is True:
+            del log_record[IS_STRUCTLOG_KEY]
+        else:
+            # Applying contextvars to non-structlog events
+            contextvars = merge_contextvars(None, None, {})
```

Comment:
Not sure to understand what this is doing

Reply:
`merge_contextvars` is a structlog processor that basically merges the contextvars with the third argument here. I'm reusing this to also append contextvars binded attributes to the non-structlog log event.

We are binding such attribute here: https://github.com/dialoguemd/covidflow/pull/261/files#diff-25f4a67a16aaa483d26997883709da28R19